LSL.py

```diff
@@ -112,16 +127,15 @@ def __collect_data():
         for real-time data processing. Uses StreamInlet.time_correction() to convert LSL to system timestamps using a
         constantly updated offset. The precision of these estimates should be below 1 ms (empirically within +/-0.2 ms).
         """
-
         while LSL.collecting:
             for stream_type, stream in LSL.streams.items():
-                data_row = {'Timestamp': None, 'Label': "" if not LSL.collection_label else LSL.collection_label}
+                data_row = {'Timestamp': None, 'Label': "Resting" if not LSL.collection_label else LSL.collection_label}
```

Comment:
I want to change it to no event instead of resting

Reply:
Added config variable